src/expressServer.js

```diff
@@ -101,7 +101,7 @@ app.use(passport.session());
 const connectDB = require('./db/connect');
 
 //routers
-const authRouter = require('./routes/authRoutes');
+const authRouter = require('./routes/authRoutes'/* #swagger.tags = ['Authentication'] */); 
```

Comment:
This comment needs to be on the `app.use` where the route is defined (line 114); not here where it is imported.

```suggestion
const authRouter = require('./routes/authRoutes'); 
```